.cirrus.yml

```diff
@@ -56,6 +56,7 @@ gcp_credentials: ENCRYPTED[!77d4c8251094346c41db63cb05eba2ff98eaff04e58c5d0e2a8e
 
 # Run a simple lint on the community cluster
 flake8_task:
+    skip: &man-changes-include "changesInclude('man/*')"
```

Comment:
Thanks for this improvement! Shouldn't be there rather `changesIncludeOnly`, per https://cirrus-ci.org/guide/writing-tasks/#supported-functions ? Since having a change in code and man pages, we should not skip the tests.

Reply:
yup, I agree, now that I re-read that statement (with a fresh morning head), it makes more sense; I'll get that updated; thanks for keeping me in check :)